Joanavel branch test

JoanavelPascual7 opened the pull request:
minor footer update

JoanavelPascual7 committed:
footer styling

JoanavelPascual7 committed:
minor footer changes

JoanavelPascual7 committed:
Merge branch 'main' into joanavel-branch-test

JenniferEinstein reviewed (approved):
Looks great. Lots of good styling changes.

JenniferEinstein committed:
Delete Videos.js

I noticed that the last line of code was accidentally deleted, so I deleted the whole thing since that was the only change.

JenniferEinstein reviewed (approved):
I deleted the video.js file that was changed because the change was an accidental error.

JenniferEinstein committed:
Merge branch 'main' into joanavel-branch-test
